tests/src/test/java/edu/snu/onyx/tests/examples/beam/BroadcastITCase.java

```diff
@@ -18,32 +18,44 @@
 import edu.snu.onyx.client.JobLauncher;
 import edu.snu.onyx.examples.beam.Broadcast;
 import edu.snu.onyx.tests.compiler.CompilerTestUtil;
+import edu.snu.onyx.tests.examples.ExampleTestUtil;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
+import java.io.IOException;
+
 /**
  * Test Broadcast program with JobLauncher.
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(JobLauncher.class)
 public final class BroadcastITCase {
   private static final int TIMEOUT = 120000;
-  private static final String input = CompilerTestUtil.rootDir + "/../examples/src/main/resources/sample_input_mr";
-  private static final String output = CompilerTestUtil.rootDir + "/../examples/src/main/resources/sample_output";
+  private static final String inputFileName = "sample_input_mr";
+  private static final String outputFileName = "sample_output";
+  private static final String testResourceFileName = "test_output_broadcast_test";
+  private static final String fileBasePath = CompilerTestUtil.rootDir + "/../examples/src/main/resources/";
+  private static final String inputFilePath =  fileBasePath + inputFileName;
+  private static final String outputFilePath =  fileBasePath + outputFileName;
 
   private static ArgBuilder builder = new ArgBuilder()
       .addJobId(BroadcastITCase.class.getSimpleName())
       .addUserMain(Broadcast.class.getCanonicalName())
-      .addUserArgs(input, output);
+      .addUserArgs(inputFilePath, outputFilePath);
 
   @Before
   public void setUp() throws Exception {
     builder = new ArgBuilder()
         .addUserMain(Broadcast.class.getCanonicalName())
-        .addUserArgs(input, output);
+        .addUserArgs(inputFilePath, outputFilePath);
+  }
+
+  private void testOutput() throws AssertionError, IOException {
```

Comment:
Do we have to create this method for each ITCase classes? We can just call the `isOutputSame` in each test.

Reply:
@sanha You are right. I will modify my code.